posthog/temporal/workflows/batch_exports.py

```diff
@@ -707,6 +717,7 @@ async def update_batch_export_backfill_model_status(inputs: UpdateBatchExportBac
 
 
 async def execute_batch_export_insert_activity(
+    destination: str,
```

Comment:
nit: You could probably get this from the Workflow context: `workflow.info().workflow_type`. But this is fine.

Reply:
> workflow_type

Ahh, I was trying to find something like this. I'll try.